src/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -40,120 +30,105 @@ import getSetting from '@salesforce/apex/RD2_entryFormController.getSetting';
 
 export default class rd2EntryForm extends LightningElement {
 
-    @api parentId;
-    @api recordId;
     listenerEvent = 'rd2EntryFormEvent';
 
     customLabels = Object.freeze({
         cancelButtonLabel,
         saveButtonLabel,
-        donationSectionHeader,
         donorSectionHeader,
         otherSectionHeader
     });
 
-    @track fields = {};
+    @api parentId;
+    @api recordId;
+
     @track record;
-    recurringDonationInfo;
+    @track contactId;
+    @track accountId;
+    @track fields = {};
     fieldInfos;
-    dayOfMonthPicklistValue;
-    lastDayPicklistValue;
 
     @track header = newHeaderLabel;
 
     @track isAutoNamingEnabled;
-    @track isMultiCurrencyEnabled;
-
-    @track contactId;
-    @track accountId;
-
-    isNew = false;
     @track isLoading = true;
-    @track isFieldInfoReady = false;
-    isRecordReady = false;
+    @track isRecordReady = false;
     isSettingReady = false;
 
     @track hasError = false;
     @track errorMessage = {};
 
-    /*******************************************************************************
+    /***
     * @description Dynamic render edit form CSS to show/hide the edit form 
     */
     get cssEditForm() {
-        return (!this.isLoading && this.isSettingReady && this.isFieldInfoReady && this.isRecordReady)
+        //Note: all of these flags need to be checked before the sections are displayed.
+        //If the isSettingReady is not checked, then the form on an error resets all fields
+        //including the Schedule section LWC fields.
+        //TODO: check why and how this expression can be simplifed.
+        return (!this.isLoading && this.isSettingReady && this.isRecordReady)
```

Comment:
When these flags are not checked (especially the `isSettingReady`), the form *resets* field values on an error.